draft-irtf-cfrg-spake2.xml

```diff
@@ -394,21 +358,16 @@ seed: edwards448 point generation seed (N)
         designed to eliminate concerns related to knowing discrete logs of M
         and N.</t>
 
-      <t>SPAKE2+ appears in <xref target="TDH" /> along with a path to a proof that
-        server compromise does not lead to password compromise under the DH assumption
-        (though the corresponding model excludes precomputation attacks).</t>
-
-      <t>Elements received from a peer MUST be checked for group membership:
-        failure to properly validate group elements can lead to attacks. Beyond the cofactor
-        multiplication checks to ensure that these elements are in the prime order subgroup
-        of G, it is essential that endpoints verify received points are members of G.</t>
+       <t>Elements received from a peer MUST be checked for group
+      membership: failure to properly validate group elements can lead
+      to attacks It is essential that endpoints verify received points
+      are members of G.</t>
 
       <t>The choices of random numbers MUST BE uniform. Randomly generated values (e.g., x and y)
           MUST NOT be reused; such reuse may permit dictionary attacks on the password.</t>
 
       <t>SPAKE2 does not support augmentation. As a result, the server has to
-        store a password equivalent. This is considered a significant drawback,
-        and so SPAKE2+ also appears in this document.</t>
+        store a password equivalent. This is considered a significant drawback.</t>
```

Comment:
I'd consider going with "a significant drawback for some use cases".  For Kerberos we basically don't care.